src/redux/actions/projects/createProject.js

```diff
@@ -0,0 +1,28 @@
+import moment from 'moment';
+import { v4 as uuidv4 } from 'uuid';
+
+import {
+  PROJECTS_CREATE,
+} from '../../actionTypes/projects';
+import { projectTemplate } from '../../reducers/projects/initialState';
+
+const createProject = (
```

Comment:
what do we do with the rest of the redux store when we have multiple projects? How do we know if, for example, the contents of differential expression in redux are relevant for the selected project, or are actually for a different project?

Reply:
The contents of for data processing, data exploration and plots and tables are actually related to experiments, not projects. So I imagine after selecting an experiment, the contents of the relevant stores will updated according to the experimentId.

Reply:
Yes, they are related to experiments, this is true. However, at the moment the contents of the redux store are not associated with an experiment, which means that if a user has viewed experiment `asd` before and decided now to view another experiment `abc`, redux will not dispatch new actions to update the data in the store. Is this something that we are planning to fix for the MVP, or are we going to support only 1:1 mapping between experiments and projects for now?

Reply:
Will have it ready for MVP in a separate ticket